frontend/consts_gen.py

```diff
@@ -375,6 +376,7 @@ def out(fmt, *args):
         out('  case %s: return %s;' % (a, b))
       out("""\
   }
+  return redir_arg_type_e::Undefined;
```

Comment:
Actually these can all be assert(0) -- I tried and that shuts up the warning about unreachable.   

So then we don't need new undefined values